packages/backend-update/src/tests/user/controller.test.ts

```diff
@@ -0,0 +1,116 @@
+import { createContainer } from '@/index'
+import { Bindings } from '@/app'
+import { env } from '@/config/env'
+import { Container } from '@/container'
+import { createApp, TestApp } from '../app'
+import { Knex } from 'knex'
+import { truncateTables } from '../tables'
+import { Request, Response } from 'express'
+import {
+  createRequest,
+  createResponse,
+  MockRequest,
+  MockResponse
+} from 'node-mocks-http'
+// import type { AuthController } from '@/auth/controller'
+import type { AuthService } from '@/auth/service'
+import { errorHandler } from '@/middleware/errorHandler'
+import { faker } from '@faker-js/faker'
+import { applyMiddleware, uuid } from '../utils'
+import { withSession } from '@/middleware/withSession'
+import type { UserService } from '@/user/service'
+import type { UserController } from '@/user/controller'
+
+describe('User Controller', (): void => {
+  let bindings: Container<Bindings>
+  let appContainer: TestApp
+  let knex: Knex
+  let authService: AuthService
+  // let authController: AuthController
```

Comment:
do we need the commented out lines in this file?

Reply:
Nope. Will remove the dead code. 👍